src/modules/map/components/openlayers/utils/styleFromLiterals.js

```diff
@@ -274,8 +274,8 @@ OlStyleForPropertyValue.prototype.setOlRotation_ = function (
 }
 
 OlStyleForPropertyValue.prototype.getOlStyle_ = function (feature, resolution, properties) {
-    // A value can be 0
-    const value = properties[this.key] ? properties[this.key] : this.defaultVal
+    // Use default value if key is not found in properties
+    const value = properties[this.key] !== undefined ? properties[this.key] : this.defaultVal
```

Comment:
What if the value is `null` do we want to have `null` as value or the default value int his case ?

Reply:
I am open for both cases. I just check the old geoadmin, it only set to `defaultVal` if the value is undefined. Should I go with the old approach or should I handle the null ?

See: https://github.com/geoadmin/mf-geoadmin3/blob/develop/src/components/StylesFromLiteralsService.js#L292

Reply:
Currently, I update it like @pakb suggests